### DISCUSSION TITLE
How do I resolve a `magit-3.0.0 unavailable` error?

### DISCUSSION THREAD

**Discussion by zachlefevre** [magit/magit#4501] [Support] [answered]
I'm getting
```
Unable to activate package ‘org-roam’.
Required package ‘magit-section-3.0.0’ is unavailable
Unable to activate package ‘magit’.
Required package ‘git-commit-20210806’ is unavailable
Unable to activate package ‘magit’.
Required package ‘git-commit-20210806’ is unavailable
Unable to activate package ‘forge’.
Required package ‘magit-3.0.0’ is unavailable
```
when I evaluate my init.el

I'm relatively new with emacs. From what I can tell this means magit can't load because git-commit cannot load.

I've checked melpa and can see these required packages. I've refreshed by packages. I'm not sure what to do next. Any guidance would be greatly appreciated.

**Comment by tarsius** [comment, marked as answer]
Do `M-x package-list-packages`. Does it list these or higher versions? If not try refreshing again. Are any errors being reported? Then something(tm) is wrong with `package.el`. Please take this support request somewhere else such as https://www.reddit.com/r/emacs/. There are many people who use `package.el` there. I don't use `package.el`.

**Comment by BenSturmfels** [comment]
For anyone else hunting around, I was seeing this error due the `(package-initialize)` in my init. Since Emacs 27 this initialisation is done automatically so should be removed from your init.
